modules/delegates/components/DelegatesSystemInfo.tsx

```diff
@@ -44,6 +49,16 @@ export function DelegatesSystemInfo({
       title: 'Total MKR delegated',
       id: 'total-mkr-system-info',
       value: new BigNumber(stats.totalMKRDelegated).toFormat(2)
+    },
+    {
+      title: 'Percent of MKR delegated',
+      id: 'percent-mkr-system-info',
+      value: totalMKR
+        ? `${new BigNumber(stats.totalMKRDelegated)
+            .dividedBy(totalMKR.toBigNumber())
+            .multipliedBy(100)
+            .toFormat(2)}%`
+        : '0'
```

Comment:
Wonder if we should have a value other than "0" when we don't have the data fetched? We used to use `--` or `n/a` or maybe put a skeleton loader there?